tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecSuite.scala

```diff
@@ -91,25 +91,29 @@ class AdaptiveQueryExecSuite
     }
   }
 
-  test("SPARK-30953: InsertAdaptiveSparkPlan should apply AQE on child plan of write commands") {
+  test("Plugin should translate child plan of write commands to GPU") {
 
     val conf = new SparkConf()
       .set(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key, "true")
       .set(SQLConf.ADAPTIVE_EXECUTION_FORCE_APPLY.key, "true")
+      // DataWritingCommandExec does not get translated to GPU because the plugin
+      // currently doesn't support the CreateDataSourceTableAsSelectCommand command
```

Comment:
This test should explicitly disable translating DataWritingCommandExec, otherwise when we fix CreateDataSourceTableAsSelectCommand to be on the GPU then this test will break.  I'm thinking something like this:
```scala
  .set("spark.rapids.sql.exec.DataWritingCommandExec", "false")
```